src/linter/ui5Types/SourceFileLinter.ts

```diff
@@ -681,37 +681,41 @@ export default class SourceFileLinter {
 		}
 
 		// There can be multiple and we need to find the right one
-		const [constructSignature] = classType.getConstructSignatures();
+		const allConstructSignatures = classType.getConstructSignatures();
```

Comment:
If `!node.arguments.length` we can skip most of this, right?

Reply:
Done